packages/ra-core/src/core/useRegisterResource.ts

```diff
@@ -0,0 +1,19 @@
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
👍 will be easier to move that logic out of the Redux store now that it's behind a custom hook